MAINTAINERS.md

```diff
@@ -88,41 +93,43 @@ A simple majority vote is required to authorize a SPIRE release at a specific co
   * Do not merge this PR yet. It will be updated later to use the real version pin rather than the commit hash.
   * If anything unusual is encountered during this process, a comment MUST be left on the release issue describing what was observed.
 
-The following steps must be completed to perform a release:
-* Mark the pull request to update the CHANGELOG as "Ready for review". Make sure that it is updated with the final release date. **At least two approvals from maintainers are required in order to be able to merge it**. If a version branch was created for the realease, cherry-pick the final CHANGELOG changes into the version branch once they are merged.
-* If releasing from main and the current state of the main branch has diverged from the candidate commit due to just the CHANGELOG changes, the candidate commit is now the one that includes the updated CHANGELOG. If releasing from a version branch, the candidate commit is now the one that has the CHANGELOG changes cherry-picked in the branch.
-* Cut an annotated tag against the release candidate named `vX.X.X`, where `X.X.X` is the semantic version number of SPIRE.
-  * The first line of the annotation should be `vX.X.X` followed by the CHANGELOG. **There should be a newline between the version and the CHANGELOG**.
+The following steps must be completed by the primary on-call maintainer to perform a release:
+* Mark the pull request to update the CHANGELOG as "Ready for review". Make sure that it is updated with the final release date. **At least two approvals from maintainers are required in order to be able to merge it**.
+* Cut an annotated tag against the release candidate named `vX.Y.Z`, where `X.Y.Z` is the semantic version number of SPIRE.
+  * The first line of the annotation should be `vX.Y.Z` followed by the CHANGELOG. **There should be a newline between the version and the CHANGELOG**. The tag should not contain the Markdown header formatting because the "#" symbol is interpreted as a comment by Git.
 * Push the annotated tag to SPIRE, and watch the build to completion.
   * If the build fails, or anything unusual is encountered, abort the release.
     * Ensure that the GitHub release, container images, and release artifacts are deleted/rolled back if necessary.
-* Visit the releases page on GitHub, copy the release notes, click edit and paste them back in. This works around a GitHub rendering bug that you will notice before completing this task.
-* Close the GitHub project created to track the release process.
+* Visit the releases page on GitHub, copy the release notes, click edit and paste them back in. This works around a GitHub Markdown rendering bug that you will notice before completing this task.
+* Create Git tags (not annotated) with the name `vX.Y.Z` in the [spire-api-sdk](https://github.com/spiffe/spire-api-sdk) and [spire-plugin-sdk](https://github.com/spiffe/spire-plugin-sdk) repositories for the HEAD commit of the main branch.
```

Comment:
this ones may have more work, we have releases we need to cherry pick commits from next, and and then we can create the tag, and it is possible we forgot to apply those changes...
So what do you think about creating a DRAFT PR 1w before release (or the same day we start with the release process) so we can be sure we get into all we need? 
and at the same time people that is watching for changes can get noticed...
Another possible approach is that any time we have a PR merged to SPIRE that required changes on SDK we merge them to main...

Reply:
Yeah this is a good point. I will make a note of making sure the spire-api-sdk and spire-plugin-sdk repositories have  all intended changes merged from `next` to `main` before the changelog PR is prepared one week prior to the release.